### PR TITLE
Functional tests.

### PR DESCRIPTION
**Pull request by YuriGizra**
#3

### CODE REVIEW

**Comment by YuriGizra** [comment]
@amitaibu , Ready for review.

**Comment by YuriGizra** [comment]
selection_322

File: behat/features/homepage.feature [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
+  @javascript
   Scenario: Visit the homepage
-    Given I am an anonymous user
-    When  I visit the homepage
-    Then  I should have access to the page
+    Given I visit the homepage
+    When  I go to log in page
+    And   I log in as registered user
+    Then  I should see the dashboard as registered user
+#    And   I should see "Gizra" in the "" element
```
Comment (amitaibu) [inline behat/features/homepage.feature]: remove

File: behat/features/homepage.feature [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+  We need to be able to have access to the dashboard
 
-  @api
+  @javascript
   Scenario: Visit the homepage
-    Given I am an anonymous user
-    When  I visit the homepage
-    Then  I should have access to the page
+    Given I visit the homepage
+    When  I go to log in page
```
Comment (amitaibu) [inline behat/features/homepage.feature]:
```
Given I visit the homepage
When I login
Then  I should see the dashboard as registered user
```

**Comment by YuriGizra** [comment]
@amitaibu , Fixed.

**Comment by YuriGizra** [comment]
@amitaibu, ready

**Pull request merged**